lib/inspec/resources/oracledb_session.rb

```diff
@@ -96,7 +96,8 @@ def command_builder(format_options, query)
       if @db_role.nil? || @su_user.nil?
         verified_query = verify_query(query)
       else
-        escaped_query = escape_query(query)
+        escaped_query = query.gsub(/\\\\/, "\\").gsub(/"/, '\\"')
```

Comment:
could this be moved into the same `escape_query` method?

Reply:
Yes, can keep the old refactoring change on this. Will do.

Reply:
Hey this is still not addressed